core/src/main/java/org/apache/iceberg/LocationProviders.java

```diff
@@ -104,9 +106,11 @@ public String newDataLocation(String filename) {
   }
 
   static class ObjectStoreLocationProvider implements LocationProvider {
-    private static final Function<Object, Integer> HASH_FUNC =
-        Transforms.bucket(Integer.MAX_VALUE).bind(Types.StringType.get());
 
+    private static final HashFunction HASH_FUNC = Hashing.murmur3_32_fixed();
+    private static final BaseEncoding BASE64_ENCODER =
+        BaseEncoding.base64Url().lowerCase().omitPadding();
```

Comment:
@singhpk234, it is a bug that I used `lowerCase` here. The build is failing because base64 encoding requires both lower and upper case letters. Can you fix the build by removing this?

Reply:
ACK removed the lowercase() from here .